lib/addons/flagger/index.ts

```diff
@@ -0,0 +1,72 @@
+import 'source-map-support/register';
+import * as blueprints from '../../../lib';
+import { Construct } from 'constructs';
+import { Values } from "../../spi";
+import merge from "ts-deepmerge";
+
+/**
+ * User provided options for the FlaggerAddonProps values.
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {//this is the root level
+  prometheusInstall?: boolean;
+  meshProvider?: MeshProviderOptions;
+  crd?: boolean;
```

Comment:
let's finalize whether we need CRDs to be installed. 
simple question: if this is not specified are CRDs installed by default?

If the answer is `yes`, let's drop it.

Reply:
That depends on the Helm version the user has. Should we keep it in order to support someone on helm v2 or should we assume all are using helm v3?